### PR TITLE
Ensure console URL is available before offering it to the user

### PR DESCRIPTION
**Pull request by andreadecorte**
As it may be not the case in HyperShift context during cluster creation

### CODE REVIEW

File: cmd/create/idp/cmd.go [hunk 1]
```diff
@@ -435,10 +435,13 @@ func doCreateIDP(
...
 		"Identity Provider '%s' has been created.\n"+
 			"   It may take several minutes for this access to become active.\n"+
-			"   To add cluster administrators, see 'rosa grant user --help'.\n"+
-			"   To login into the console, open %s and click on %s.",
-		idpName, cluster.Console().URL(), idpName,
-	)
+			"   To add cluster administrators, see 'rosa grant user --help'.\n", idpName)
+	// Console may not be available yet
+	clusterConsole := cluster.Console().URL()
+	if clusterConsole != "" {
```
Comment (zgalor) [inline cmd/create/idp/cmd.go:441]: I think `cluster.Console()` can also return nil so I suggest
```
if cluster.Console() != nil && cluster.Console().URL() != "" {
	clusterConsole := cluster.Console().URL()
...
}
```

Comment (andreadecorte) [reply cmd/create/idp/cmd.go]: fixed, thanks

**Pull request merged**